content/building/yaml.md

```diff
@@ -108,33 +109,80 @@ You can use `codemagic.yaml` to define several workflows for building a project.
         triggering:
         branch_patterns:
         scripts:
-        publishing:
         artifacts:
+        publishing:
 
 The main sections in each workflow are described below.
 
 ### Environment
 
-`environment:` Contains your environment variables and enables to specify the version of Flutter used for building. Make sure to [encrypt the values](#encrypting-sensitive-data) of variables that hold sensitive data. 
+`environment:` Contains your environment variables and enables to specify the version of Flutter used for building. This is also where you are required to add credentials and API keys required for code signing. Make sure to [encrypt the values](#encrypting-sensitive-data) of variables that hold sensitive data. 
```

Comment:
> This is also where you are required to add credentials and API keys required for code signing.

It is not _required_ to add the credentials here. Each property can be specified as a command argument to programs with dedicated flags.

As an example the following yield exactly the same result:
```bash
app-store-connect fetch‑signing‑files --issuer-id=<ISSUER-ID> --key-id=<KEY-IDENTIFIER> --private-key=@file:/path/to/app-store-connect-private-key.p8 --certificate-key=@file:/path/to/certificate-private-key.pem
``` 
and
```bash
APP_STORE_CONNECT_ISSUER_ID=<ISSUER-ID>
APP_STORE_CONNECT_KEY_IDENTIFIER=<KEY-IDENTIFIER>
APP_STORE_CONNECT_PRIVATE_KEY=<app-store-connect-private-key-contents-as-pem>
CERTIFICATE_PRIVATE_KEY=<code-signing-certificate-private-key-contents-as-pem>

app-store-connect fetch‑signing‑files
``` 

The environment variables are just shortcuts / default fallbacks to avoid cluttering the script phases. There are number of ways those values can be specified as stated [here](https://github.com/codemagic-ci-cd/cli-tools/blob/master/docs/app-store-connect/fetch%E2%80%91signing%E2%80%91files.md#--issuer-idissuer_id).